src/panel_material_ui/widgets/FileInput.jsx

```diff
@@ -3,6 +3,7 @@ import {styled} from "@mui/material/styles"
 import CloudUploadIcon from "@mui/icons-material/CloudUpload"
```

Comment:
When I try to upload a csv file via example below I only see

![image](https://github.com/user-attachments/assets/08b98bdf-fd96-4a8c-b7e8-f5b23d0fb7aa)

The value never gets to the Python side.

```python
import panel_material_ui as pmui
import panel as pn
import io
import pandas as pd

pn.extension()

def csv_view(value):
    """Callback function to handle csv upload."""
    if value:
        value = value.encode("utf-8")
        df = pd.read_csv(io.BytesIO(value))
        return pn.pane.Tabulator(df)
    return "No Table"

fileinput = pmui.FileInput(
    label="Upload a CSV file",
    accept=".csv",
    color="success",
    variant="outlined",
)

pn.Column(fileinput, pn.bind(csv_view, value=fileinput)).servable()
```

Reply:
Solved partially by adding logging about issue on server and in browser console.

I think a tooltip is needed on the client side to explain to the user the error or what to do.

Reply:
I've added a tooltip on the client side.